staging/src/k8s.io/kube-controller-manager/config/v1alpha1/doc.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Note: The referenced generic ComponentConfig packages with conversions
+// between the types (e.g. the external package) needs to be given as an
+// input to conversion-gen for it to find the native conversation funcs to
+// call.
+
+// +k8s:deepcopy-gen=package
+// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/componentconfig
```

Comment:
Remove this, there should be no dependency on `pkg/apis/componentconfig` which we'll remove